src/Documents.ts

```diff
@@ -220,6 +220,40 @@ class Documents implements Pointer.Library, Pointer.Validator, ObjectSchema.Reso
 		});
 	}
 
+	getChildren( parentURI:string, requestOptions:HTTP.Request.Options = {} ): Promise<[ Pointer.Class[], HTTP.Response.Class ]> {
+		if( !! this.context ) parentURI = this.context.resolve( parentURI );
+		if ( this.context && this.context.auth.isAuthenticated() ) this.context.auth.addAuthentication( requestOptions );
+
+		let containerRetrievalPreferences:HTTP.Request.ContainerRetrievalPreferences = {
+			include: [
+				NS.LDP.Class.PreferContainment,
```

Comment:
NS.LDP.Class.PreferContainment is the same as NS.C.Class.PreferContainmentTriples. Please only use PreferContainment.

NS.LDP.Class.PreferMembership is also the same as NS.C.Class.PreferMembership